yarn.lock

```diff
@@ -1921,7 +1914,7 @@ __metadata:
   languageName: unknown
   linkType: soft
 
-"@carbon/icon-helpers@npm:^10.28.0, @carbon/icon-helpers@npm:^10.49.0, @carbon/icon-helpers@workspace:packages/icon-helpers":
```

Comment:
This is the source of the invalid resolution: `@carbon/icon-helpers@npm:^10.28.0`